WinCompatibilityPack/WinCompatibilityPack.psm1

```diff
@@ -505,6 +508,11 @@ function Import-WinModule
             $PassThru
     )
 
+    if ($PSEdition -eq "Desktop")
+    {
+        throw "This command is only supported on PowerShell Core"
```

Comment:
We should consider using ThrowTerminationError and an appropriate exception type, e.g.:

```powershell
    $e = [System.Management.Automation.ErrorRecord]::new(
        [System.PlatformNotSupportedException]::new(
            "This command is only supported on PowerShell Core"
        ),
        'DesktopPlatformNotSupported',
        'NotImplemented',
        $null
    )
    $PSCmdlet.ThrowTerminatingError($e)
```